### ISSUE TITLE
Test prediction score calculator for pre-training datasets

### ISSUE THREAD

**Issue by KonstantinWilleke** [sinzlab/sensorium#81]
As a demo for when the competition goes live, it would be great to have a way to test the model predictions on the test set of one of the pre-training scans, without needing the submission + ground truth file.
